### ISSUE TITLE
Scoping or filtering by acts_as_taggable_on tags

### ISSUE THREAD

**Issue by rilla** [activeadmin/activeadmin#520]
Hi there, 

Given this model:

``` ruby
class Question < ActiveRecord::Base
  acts_as_taggable_on :tags
end
```

I'd like to be able to filter/scope questions by `tag.name`.

Maybe I'm missing something very obvious but I can't find a way to implement this. Any ideas? 

Thanks!

**Comment by scarver2** [comment]
Thank you @EricLondon. Here is the same code in Ruby 1.9+ syntax with concise mapping. 
Note: I added case-insensitive sorting to the end

```ruby
filter :taggings_tag_name, as: :check_boxes, collection: -> { Movie.genre_counts.map{&:name}.sort_by(&:downcase) }
```

**Comment by caifara** [comment]
At least I'm not the only one searching on this. This is what I came up with, given some new features and known problems of ransack. I adapted my solution to the given context. (also: [chosen.js](http://staal.io/blog/2013/02/26/mastering-activeadmin/) can make the UX somewhat better in my opinion)

activeadmin filter:
```` ruby
  filter :ransack_tagged_with, 
    as: :check_boxes, 
    collection: ->{ Movie.genre_counts.map{&:name}.sort_by(&:downcase) }
```

model:
````ruby
  scope :ransack_tagged_with, ->(*tags){ tagged_with(tags) }

  def self.ransackable_scopes(auth=nil)
    %i(ransack_tagged_with)
  end
```

* setting `tagged_with` itself in the ransackable_scopes doesn't work because of this issue: https://github.com/activerecord-hackery/ransack/issues/404
* this method is easily expandable to multiple tag lists
